src/PhpFlo/Fbp/Tests/FbpDefinitionTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
```

Comment:
weird, is it just for me that this file isn't highlighted in the review page? https://github.com/phpflo/phpflo/pull/38/files#diff-2cc6896b59caf9159e948b1004a5c344R3